File: 2.0/python/contacts/getContactList.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.contacts.getContactList(
    id=379506003,
    # return only specific fields
    fields='id,name,size,created'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/contacts/getContactList.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->getContactList();
        $request->getOperationConfig()->setPathParameters(array("id" => 379506003));
        $request->getOperationConfig()->setQueryParameters(array("fields" => "id,name,size,created"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
